sql/MovieTicketApp.sql

```diff
@@ -10,31 +10,54 @@ create table users(
     CONSTRAINT users_pkey PRIMARY KEY (id)
 );
 
-create table movies(
-	id  serial NOT NULL ,
-	movie_name character varying(40) NOT NULL,
-    actor_name character varying(40) NOT NULL,
-    rating real NOT NULL,
-    status character varying(40) NOT NULL,
-    CONSTRAINT movies_pkey PRIMARY KEY (id)
-    screen character varying(40)  NOT NULL,
-    CONSTRAINT screen FOREIGN KEY (screen)
-        REFERENCES screens (name) MATCH SIMPLE
-    
-);
+create table screens(id serial primary key,
+					  name varchar(40) not null,
+					  status varchar(40) not null default 'INACTIVE',
```

Comment:
Add check constraint for status column.

CHECK ( status in ('ACTIVE','INACTIVE') );